### ISSUE TITLE
Google analytics

### ISSUE THREAD

**Issue by aserawayneasera** [gbif/hp-sprep#14]
Where can I place tags for google analytics please?

**Comment by MortenHofft** [comment]
This is blocked by https://github.com/gbif/hosted-portals/issues/144
Once that is in place I will add instructions to the documentation and send you an update. If I forget, then please poke me. But I won't do anything until above issue is resolved.

**Comment by aserawayneasera** [comment]
Ok thanks. I will leave this open then until it is unblocked.

**Comment by MortenHofft** [comment]
Hi Wayne
The theme now supports a simple way to add google analytics. See below links

https://hp-theme.gbif-staging.org/measurements
https://hp-theme.gbif-staging.org/privacy-policy-template
https://github.com/gbif/hp-sprep/blob/master/_config.yml#L57

~The terms are not ready yet, so it isn't ready for production yet.~

**Comment by aserawayneasera** [comment]
Thanks Morten. I already set this up.
